guides/common/modules/proc_synchronizing-an-external-smart-proxy.adoc

```diff
@@ -1,4 +1,4 @@
-[id="Synchronizing_an_External_Smart_Proxy_{context}"]
+[id="Synchronizing_an_External_{smart-proxy-context}_{context}"]
```

Comment:
Not sure about the level of uggliness of this :D

Reply:
I think that with the use of context attributes in anchors it would be better to rewrite the entire anchor in lowercase. I would also use `-` instead of `_` (except for the `_{context}` part), because that is what context attributes use. This already seems to be the case with anchors that use lowercase letters, as can be seen with 
```
git grep -E '\[id="[a-z]'
```

Reply:
Agreed.